CHANGES.rst

```diff
@@ -6,7 +6,8 @@ There's a frood who really knows where his towel is.
 2.1b2 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
-- Nothing changed yet.
+- Fix upgrade process between versions 1.0 and 2.0 (fixes `#198`_).
```

Comment:
Mainly a "cosmetic" change, I would put "Fix upgrade process between versions 1.0 and 2.0. Check README.rst documentation for 1.x to 2.x migration. (fixes `#198`_)."